m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -2095,25 +2093,35 @@ PROCEDURE BUILDSLTRANSFERLIST
          TRANCREATEDATE(TRANCOUNT)=TRANSFERCREATEDATE
          TRANCREATEUSER(TRANCOUNT)=TRANSFERCREATEUSER
 
+         BNODEBUGMSG=FORMAT("**** TESTING FOR READONLY DATA - SHARE TRANSFER LOC ########9",TARGETTRANSFERLOC)
+         CALL BNODEBUGMSGADD
+         BNODEBUGMSG=FORMAT("EXTRECFOUND:9  ",EXTRECFOUND)+
+                     FORMAT("PARAMALLOWEDIT:9  ",PARAMALLOWEDIT)+
+                    "   SHARE TRAN FREQ:"+
+                     TRANSFERFREQCHR(TRANFREQALT,SHARE TRANSFER:FREQUENCY)+
+                    FORMAT("  TRANSFERCREATEUSER:,###9+  ",TRANSFERCREATEUSER)+
+                    FORMAT("USERSWHOCANEDIT 9999",TRANSFERCREATEUSER)+
+                    FORMAT("=9",USERSWHOCANEDIT(TRANSFERCREATEUSER))
+         CALL BNODEBUGMSGADD
+
 [* Determine if existing transfer is editible or not
 *]
-         IF ((EXTRECFOUND<>FALSE AND TRANCREATEUSER(TRANCOUNT)>=800) OR
-             PARAMALLOWEDIT=TRUE) AND
-             TRANSFERFREQCHR(TRANFREQALT,SHARE TRANSFER:FREQUENCY)<>"" THEN
-          TRANREADONLYSTATUS(TRANCOUNT)="false"
-         ELSE
-          TRANREADONLYSTATUS(TRANCOUNT)="true"
+         IF TRANSFERFREQCHR(TRANFREQALT,SHARE TRANSFER:FREQUENCY)<>"" AND
+           (PARAMALLOWEDIT=ALLOWEDITTRUE OR
+           (PARAMALLOWEDIT=ALLOWEDITUSER AND USERSWHOCANEDIT(TRANSFERCREATEUSER)=TRUE) OR
+            TRANSFERCREATEUSER=SYMXUSERNUMBER) THEN
+             TRANREADONLYSTATUS(TRANCOUNT)="false"
+            ELSE
+             TRANREADONLYSTATUS(TRANCOUNT)="true"
 
          TRANACCOUNTNAME(TRANCOUNT)=SHARE:NICKNAME
          IF SHARE:NICKNAME="" THEN
-          TRANACCOUNTNAME(TRANCOUNT)=SHARE:DESCRIPTION
-
+         TRANACCOUNTNAME(TRANCOUNT)=SHARE:DESCRIPTION
```

Comment:
GitHub needs a space right before TRANACCOUNTNAME to line up with the IF statement

Reply:
@jordan-keenan Corrected in next commit